src/lib/core/components/visualizations/VisualizationsContainer.tsx

```diff
@@ -328,7 +336,10 @@ function FullScreenVisualization(props: Props & { id: string }) {
               }
             />
           </h3>
-          <div className="Subtitle">{overview?.displayName}</div>
+          {/* add viz description */}
+          <div className="Subtitle">
+            {overview?.displayName}: {overview?.description}
+          </div>
```

Comment:
We agreed to revert this today.

Reply:
thank you for reminding me and removed :)